Gopkg.toml

```diff
@@ -3,7 +3,7 @@
 # Refer to https://golang.github.io/dep/docs/Gopkg.toml.html
 # for detailed Gopkg.toml documentation.
 #
-# required = ["github.com/user/thing/cmd/thing"]
+# required = ["github.com/litmuschaos/chaos-operator"]
```

Comment:
This change is not required, Can you also remove the commented parts from here just keep the line 1 to 5